src/option/option-window.tsx

```diff
@@ -100,18 +119,25 @@ export default class OptionWindow extends React.Component {
 	handleSubmit = (event) => {
 		event.preventDefault();
 		let flags: string[];
+		let authors: string[];
 		let frequency: number;
 		let repoUrl: string;
+		let ExcludeOrOnly: boolean;
+
 		frequency = event.target.elements.frequencyInput.value;
 		flags = event.target.elements.flagInput.value.split(';').map(item => item.trim()).filter((e) => e);
+		authors = event.target.elements.authorInput.value.split(';').map(item => item.trim()).filter((e) => e);
 		repoUrl = event.target.elements.urlInput.value;
+		ExcludeOrOnly = event.target.elements.ExcludeOrOnly.value === 'Exclude' ? false : true;
```

Comment:
Why not `event.target.elements.ExcludeOrOnly.value !== 'Exclude'`?
Also if this is a boolean, it'd be better named for what it is when true, as it's not clear which is which at the moment.